systemtests/setup_tests.py

```diff
@@ -27,44 +27,42 @@ def setup_tests(ctx):
         os.makedirs(ZIPS_DIR)
 
     click.echo("Generating systemtest data files ...")
-    try:
```

Comment:
I removed the try block here. If this fails, we want to know about it and get a traceback so we can either run it again or improve the script.